lib/entities/organization-space-membership.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Organization space membership instances
+ * @namespace OrganizationSpaceMembership
+ */
+
+import cloneDeep from 'lodash/cloneDeep'
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import enhanceWithMethods from '../enhance-with-methods'
+
+/**
+ * @memberof OrganizationSpaceMembership
```

Comment:
There isn't an OrganizationSpaceMembership entity in Contentful. What changes is the scoping.
`organizations/:orgId/space_memberships` will just return `SpaceMembership` objects, but across all spaces in the org.
So I would just remove these type definitions.

Reply:
yeah, you’re totally right.. I removed them. thx for feedback 🙂